src-built-in/preloads/zoom.js

```diff
@@ -2,38 +2,138 @@
 // This global will contain our current zoom level
 window.fsblZoomLevel = 1;
 
+// Global variable with the timeout for the zoom
+window.zoomTimeout = 1000;
```

Comment:
I'd like to add config for this too, but I'm not sure where that should be.

Reply:
NVM, looks like I already did it (see zoomConfigHandler). I put it under Window Manager.